cerberus/client.py

```diff
@@ -111,6 +289,56 @@ def get_secrets(self, vault_path):
 
         return secret_resp.json()
 
+    def list_secrets(self, vault_path):
+        """Returns json secrets based on the vault_path, this will list keys in a folder"""
+        secret_resp = requests.get(self.cerberus_url + '/v1/secret/' + vault_path + '?list=true',
+                                   headers=self.HEADERS)
+        self.throw_if_bad_response(secret_resp)
+        return secret_resp.json()
+
+    def put_secret(self, vault_path, key, merge=True):
```

Comment:
Not sure what `key` is. Seems like it's more of a data map or secrets data object?